package.json

```diff
@@ -11,66 +18,106 @@
   ],
   "author": {
     "name": "Derek Brans",
-    "email": "dbrans@gmail.com",
     "web": "http://github.com/dbrans"
   },
   "contributors": [
     {
       "name": "Derek Brans",
-      "email": "dbrans@gmail.com",
-      "web": "http://github.com/dbrans"
+      "web": "https://github.com/dbrans"
     },
     {
       "name": "Nick Sullivan",
-      "email": "nick@sullivanflock.com",
       "web": "http://creationeer.me"
     },
     {
       "name": "Joshua Newman",
-      "email": "joshua.newman@krux.com"
+      "web": "https://github.com/jnewman"
+    },
+    {
+      "name": "Seth Yates",
+      "web": "https://github.com/sethyates"
     }
   ],
   "maintainers": [
     {
       "name": "Joshua Newman",
-      "email": "joshua.newman@krux.com"
+      "web": "https://github.com/jnewman"
     }
   ],
-  "bugs": "https://github.com/krux/postscribe/issues",
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/krux/postscribe/blob/master/LICENSE"
-    }
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:krux/postscribe.git"
-  },
+  "main": "dist/postscribe.js",
+  "browser": "dist/postscribe.min.js",
```

Comment:
The browser build used by Webpack and Browserify should not be minified. This makes debugging harder for users of Postscribe when they have their build system in unminified mode. Users of Postscribe with Webpack/Browserify will usually minify the complete bundle at once.